assets/l10n/app_en.arb

```diff
@@ -27,18 +31,290 @@
+    "description": "Hint text for content input when sending a message to yourself."
+  },
+  "composeBoxDMHintNoName": "Type a message",
+  "@composeBoxDMHintNoName": {
+    "description": "Hint text for content input when sending a message to one other person but we cannot determine their name."
+  },
+  "composeBoxDMHint": "Message @{user}",
+  "@composeBoxDMHint": {
+    "description": "Hint text for content input when sending a message to one other person.",
+    "placeholders": {
+      "user": {
+        "type": "String",
+        "example": "stream name"
+      }
+    }
+  },
+  "composeBoxGroupDMHint": "Message group",
+  "@composeBoxGroupDMHint": {
+    "description": "Hint text for content input when sending a message to a group."
+  },
+  "composeBoxStreamContentHint": "Message #{stream} > ${topic}",
+  "@composeBoxStreamContentHint": {
+    "description": "Hint text for content input when sending a message to a stream",
+    "placeholders": {
+      "stream": {
+        "type": "String",
+        "example": "stream name"
+      },
+      "topic": {
+        "type": "String",
+        "example": "topic name"
+      }
+    }
+  },
+  "composeBoxSendTooltip": "Send",
+  "@composeBoxSendTooltip":  {
+    "description": "Tooltip for send button in compose box."
+  },
+  "composeBoxUnknownStreamName": "(unknown stream)",
+  "@composeBoxUnknownStreamName": {
+    "description": "Replacement name for stream when it cannot be found in the store."
+  },
+  "composeBoxTopicHintText": "Topic",
+  "@composeBoxTopicHintText":  {
+    "description": "Hint text for topic input widget in compose box."
+  },
+  "composeBoxUploadingFilename": "Uploading {filename}...",
+  "@composeBoxUploadingFilename": {
+    "description": "Label in compose box showing the specified file is currently uploading.",
+    "placeholders": {
+      "filename": {
+        "type": "String",
+        "example": "file.txt"
+      }
+    }
+  },
+  "contentValidationErrorTooLong": "Message length shouldn't be greater than 10000 characters.",
+  "@contentValidationErrorTooLong": {
+    "description": "Content validation error message when the message is too long."
+  },
+  "contentValidationErrorEmpty": "You have nothing to send!",
+  "@contentValidationErrorEmpty": {
+    "description": "Content validation error message when the message is empty."
+  },
+  "contentValidationErrorQuoteAndReplyInProgress": "Please wait for the quotation to complete.",
+  "@contentValidationErrorQuoteAndReplyInProgress": {
+    "description": "Content validation error message when a quotation has not completed yet."
+  },
+  "contentValidationErrorUploadInProgress": "Please wait for the upload to complete.",
+  "@contentValidationErrorUploadInProgress": {
+    "description": "Content validation error message when attachments have not finished uploading."
+  },
+  "dialogButtonOK": "OK",
+  "@dialogButtonOK": {
+    "description": "Button label in generic dialogs to confirm."
```

Comment:
The actual use of this is somewhat more specific — it's in an *error* dialog, as the only available action, to acknowledge the error and dismiss the dialog.

I think that's a relevant distinction: it's fairly likely that if we had a dialog box with an actual choice, with options like perhaps "OK" / "Cancel", then in some languages the "OK" would be translated differently from the "OK" of the error dialog.